renovate.json

```
@@ -1,4 +1,4 @@
-{
+{ 
```

remove trailing spaces

```suggestion
{
```